cmd/root_test.go

```diff
@@ -286,6 +286,15 @@ func TestNewCommandArguments(t *testing.T) {
 				}},
 			}),
 		},
+		{
+			desc: "using the unix socket path query param",
+			args: []string{"proj:region:inst?unix-socket-path=/path/to/dir/"},
```

Comment:
nit: remove the trailing slash to make it clear we're not passing a directory.

Reply:
Done.